app/controllers/application_controller.rb

```diff
@@ -0,0 +1,2 @@
+class ApplicationController < ActionController::Base
```

Comment:
Whats this file about?

Reply:
`ApplicationController` sits at the top level of all your application's controllers which will be inherited by the other controllers. Thus, we can define methods that are to be shared across all other controllers here.